include/deal.II/dofs/block_info.h

```diff
@@ -64,16 +64,21 @@ namespace hp
  * The most common usage for this object is initializing vectors as in the
  * following code:
  *
- * <code> MGDofHandler<dim> dof_handler(triangulation);
- * dof_handler.distribute_dofs(fesystem);
+ * <code>
+ * DoFHandler<dim> dof_handler(triangulation);
+ * dof_handler.distribute_dofs(fe_system);
```

Comment:
After the removal of MGDoFHandler, you actually need to add the line
```
  dof_handler.distribute_mg_dofs(fe_system);
```
to ensure that it actually builds MG dofs as well.